pallets/frequency-tx-payment/src/capacity_stable_weights.rs

```diff
@@ -223,54 +224,70 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(4_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
-	/// Storage: Msa PublicKeyToMsaId (r:1 w:0)
-	/// Proof: Msa PublicKeyToMsaId (max_values: None, max_size: Some(48), added: 2523, mode: MaxEncodedLen)
-	/// Storage: Schemas Schemas (r:1 w:0)
-	/// Proof Skipped: Schemas Schemas (max_values: None, max_size: None, mode: Measured)
-	/// Storage: unknown `0xbd1557c8db6bd8599a811a7175fbc2fc6400` (r:1 w:1)
-	/// Proof Skipped: unknown `0xbd1557c8db6bd8599a811a7175fbc2fc6400` (r:1 w:1)
-	/// The range of component `s` is `[1, 5121]`.
-	fn apply_item_actions_with_signature(s: u32) -> Weight {
+	/// Storage: `Msa::PublicKeyToMsaId` (r:1 w:0)
+	/// Proof: `Msa::PublicKeyToMsaId` (`max_values`: None, `max_size`: Some(48), added: 2523, mode: `MaxEncodedLen`)
+	/// Storage: `Schemas::SchemaInfos` (r:1 w:0)
+	/// Proof: `Schemas::SchemaInfos` (`max_values`: None, `max_size`: Some(15), added: 2490, mode: `MaxEncodedLen`)
+	/// Storage: UNKNOWN KEY `0xbd1557c8db6bd8599a811a7175fbc2fc6400` (r:1 w:1)
+	/// Proof: UNKNOWN KEY `0xbd1557c8db6bd8599a811a7175fbc2fc6400` (r:1 w:1)
+	/// The range of component `s` is `[1024, 5120]`.
+	fn apply_item_actions_with_signature_v2_add(s: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `33377`
-		//  Estimated: `45752`
-		// Minimum execution time: 165_956_000 picoseconds.
-		Weight::from_parts(158_947_612, 45752)
```

Comment:
I'm not sure what is the pre-condition or requirements for changing stable-weights but we should know about the consequences before committing these changes

Reply:
Thanks for catching this; I meant to add a note myself but forgot.

What I found with the stable weights is that when these extrinsics were initially added, the stable weights re-used the weights from the previous version of the extrinsic. Not sure if it as a cut/paste error, or if that was intentional.

If it was intentional, the issue is that those weights would need to be copied now.
If it was not intentional, we should discuss the appropriate approach at this point.

Reply:
@wilwade we'll need to settle this question before this can be merged

Reply:
@JoeCap08055 it was intentional. They should use the same weights as before.

We try to make even new versions of the extrinsics that have the same purpose (without new features for the provider) to maintain the same cost in capacity. (Not exactly as parameter changes the cost some as well.)

Reply:
Reverted.